oak/server/oak_grpc_node.h

```diff
@@ -29,12 +29,14 @@ class OakGrpcNode final : public Application::Service {
   grpc::Status Start();
   grpc::Status Stop();
 
-  void SetUpGrpcChannels(std::shared_ptr<MessageChannel> grpc_in,
-                         std::shared_ptr<MessageChannel> grpc_out);
+  // Add the inbound channel (with implicit port name "response").
+  void AddReadChannel(std::unique_ptr<MessageChannelReadHalf> rsp_half);
```

Comment:
(suggestion) `SetResponseChannel`, and similarly elsewhere (even `LoggingNode::SetLogChannel`).

Reply:
These were named for similarity with `AddNamedChannel`, and will hopefully get removed in favour of that in a later set of commits.